xformers/factory/block_factory.py

```diff
@@ -100,6 +100,8 @@ class xFormerBlockConfig:
     layer_norm_style: LayerNormStyle
     layer_position: LayerPosition
     use_triton: bool
+    reversible: bool 
+    num_layers: int
```

Comment:
agree, feels like this is the right place

Reply:
thinking about it, it also means that some of the tests with respect to reversibility can be removed, since this will mean that all the layers in the same block will have the same characteristic